adapters/33across/33across.go

```diff
@@ -81,63 +117,64 @@ func (a *TtxAdapter) makeRequest(request *openrtb.BidRequest) (*adapters.Request
 		Uri:     a.endpoint,
 		Body:    reqJSON,
 		Headers: headers,
-	}, errs
+	}, nil
 }
 
-// Mutate the request to get it ready to send to ttx.
-func preprocess(request *openrtb.BidRequest) error {
-	var imp = &request.Imp[0]
+func makeImps(imp openrtb.Imp) (openrtb.Imp, error) {
+	impCopy := imp
```

Comment:
This line isn't necessary. The copy is made implicitly by passing by value to the func.